app/(afterLogin)/(okrplanning)/okr/settings/check-in-rule/check-in/index.tsx

```
@@ -0,0 +1,948 @@
+              {workScheduleDays.length > 0 && (
+                <Form.Item label="Applicable Days" name="applicableDays">
+                  <div className="bg-gray-50 border border-gray-200 rounded-lg p-3 space-y-2">
+                    <div className="text-xs text-gray-500 mb-1">
+                      Select days and set times
+                    </div>
+                    
+                    {/* Time Labels Header */}
+                    <div className="flex items-center justify-end pr-3">
+                      <div className="flex items-center space-x-2">
+                        <div className="w-20 text-center">
+                          <span className="text-xs text-gray-600 font-medium">Start</span>
```

ESLint

[error] Replace `Start` with `⏎····························Start⏎··························`

(prettier/prettier)